plugins/formal-verification/formver.viper/src/org/jetbrains/kotlin/formver/viper/ast/PermExp.kt

```diff
@@ -5,14 +5,13 @@
 
 package org.jetbrains.kotlin.formver.viper.ast
 
-import org.jetbrains.kotlin.formver.viper.IntoSilver
-
-sealed class PermExp : IntoSilver<viper.silver.ast.PermExp> {
+sealed class PermExp : Exp {
```

Comment:
Why is this change necessary?  I can't immediately see where you use it.

Reply:
you are right. I don't know why, but I thought it was necessary for `PredicateAccessPredicate` creation